packages/web-app-files/src/extensions.ts

```diff
@@ -19,11 +20,13 @@ export const extensions = ({ applicationConfig }: ApplicationSetupOptions) => {
   const { actions: showSharesActions } = useFileActionsShowShares()
   const { actions: quickLinkActions } = useFileActionsCopyQuickLink()
 
-  const panels = sideBarPanels()
+  const panels = useSideBarPanels()
+  const folderViews = useFolderViews()
 
   return computed(
     () =>
       [
+        ...unref(folderViews),
```

Comment:
folderViews are not a ref, but just an array... looks like that's ok, but you don't need to `unref` the var then. The sidebar panels are built as a `computed`, but looking at the code I think we could omit that and also just return an array. And also not `unref` it here then.

Reply:
okay, I'm fixing the new code, but won't touch the other code for now to avoid breaking anything while thoughtlessly changing something 😅